### PR TITLE
Problem: dirty check github action plugin don't work anymore

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- use a simpler solution

verified: https://github.com/crypto-org-chain/cronos/runs/6649208587?check_suite_focus=true

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -80,7 +80,10 @@ jobs:
         run: nix run -f ./nix gomod2nix
       - name: check working directory is clean
         id: changes
-        uses: UnicornGlobal/has-changes-action@v1.0.11
```
Comment (thomas-nguy) [inline .github/workflows/build.yml:83]: it seems the command used is pretty simple 
https://github.com/UnicornGlobal/has-changes-action/blob/v1.0.11/entrypoint.sh

Is it what you are trying to do?

Comment (yihuang) [reply .github/workflows/build.yml:83]: yeah, similar, but `git diff` can show the actual differences

Comment (yihuang) [reply .github/workflows/build.yml:83]: the github action doesn't work for some weird error, you can see in the old run, it has been malfunctioning for a while I think.

Comment (yihuang) [reply .github/workflows/build.yml:83]: but `git diff` don't check for untracked files, but I guess it's fine for our use cases.

**Review by thomas-nguy — APPROVED** [review]
lgtm

**Pull request merged**